common-api/src/main/java/com/koval/resolver/common/api/auth/CredentialsKeeper.java

```diff
@@ -43,13 +44,16 @@ public void store(Credentials credentials) {
         throw new CredentialException("Could not create credentials file: " + credentialsFile.getAbsolutePath(), e);
       }
     }
-    try (PrintWriter fileWriter = new PrintWriter(credentialsFile, charset.name())) {
-      fileWriter.println(protector.encrypt(credentials.getUsername()));
-      fileWriter.println(protector.encrypt(credentials.getPassword()));
+    try (PrintWriter printWriter = new PrintWriter(credentialsFile, charset.name());
+            BufferedWriter fileWriter = new BufferedWriter(printWriter)) {
+      fileWriter.write(protector.encrypt(credentials.getUsername()));
```

Comment:
PrintWriter.println() method add "\n" at the end of the string, but in your implementation there's no line wrapping. Username and password should be on separated lines.

Reply:
Will add